naxsi_src/naxsi_runtime.c

```diff
@@ -1133,9 +1133,10 @@ naxsi_create_log_array(ngx_http_request_ctx_t* ctx,
   sz_left -= sub;
   offset += sub;
 
-  sub = NAXSI_REQUEST_ID_SIZE << 1;
+  char* req_id = naxsi_request_id(r);
+  sub          = 32;
   if (sz_left > (100 + sub)) {
```

Comment:
add check on req_id

Reply:
And what to do based on the results of this check?

Reply:
just omit the string. (the id will be empty)